### DISCUSSION TITLE
How to make an addon work with dynamic enable/disable?

### DISCUSSION THREAD

**Discussion by mybearworld** [ScratchAddons/ScratchAddons#5961] [Help] [answered]
I have an addon where I think would be pretty easy to add dynamic enable/disable functionality to it, but I'm not sure how. I can't find it in the docs for some reason.

**Comment by rgantzos** [comment]
https://scratchaddons.com/docs/reference/addon-manifest/#dynamicdisable

**Reply by rgantzos** [reply]
(The dynamic enable is right below it)

**Reply by mybearworld** [reply]
That will just disable all CSS, but JavaScript can obviously not be undone just like that. Addons like dark-www are still able to dynamic enable/disable, even though they have JavaScript in them. Is there a way to do that somehow?

**Reply by WorldLanguages** [reply, marked as answer]
You're looking for https://scratchaddons.com/docs/reference/addon-api/addon.self/#using-addonselfdisabled-and-events-dynamicdisable

**Reply by mybearworld** [reply]
Thank you!
